src/main/java/practice/StreamPractice.java

```diff
@@ -50,7 +69,10 @@ public List<Person> selectMenByAge(List<Person> peopleList, int fromAge, int toA
      */
     public List<Person> getWorkablePeople(int fromAge, int femaleToAge,
                                           int maleToAge, List<Person> peopleList) {
-        return Collections.emptyList();
+        return  peopleList.stream()
+                .filter(p -> p.getSex() == Person.Sex.MAN ? p.getAge() >= fromAge && p.getAge() <= maleToAge :
+                        p.getAge() >= fromAge && p.getAge() <= femaleToAge)
```

Comment:
Also separate long ternary conditions with a new line, like 
```suggestion
condition 
? true
: false
```